recipe/meta.yaml

```diff
@@ -21,14 +21,15 @@ requirements:
   build:
     - toolchain
     - pkg-config  # [not win]
+    - cmake  # [win]
```

Comment:
cmake can't be a depenency here, can it?

Reply:
duh, right

Reply:
Triggered a new build without it. Must have been working on two different things at the same time.